### PR TITLE
regress: webui bsock class tls connection test

### PR DESCRIPTION
**Pull request by fbergkemper**
Test if our PHP BSock class is able to make a TLS connection with the director using certificates.

### CODE REVIEW

File: webui/tests/regress/webui-bsock-connection-test-tls.in [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+#
+# Run a simple TLS connection test
+# with an instance of the webui
+# PHP BareosBSock class
+#
+TestName="webui-bsock-connection-test-tls"
+
+PHP=@PHP@
```
Comment (arogge) [inline webui/tests/regress/webui-bsock-connection-test-tls.in]: the values should be quoted

File: webui/tests/regress/webui-bsock-connection-test-tls.in [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+scripts/cleanup
+
+copy_configs
+
+start_test
+
+run_bareos
+
+cd ${WEBUI_DIR}
```
Comment (arogge) [inline webui/tests/regress/webui-bsock-connection-test-tls.in]: variable usage should be quoted

File: webui/tests/regress/webui-bsock-connection-test-tls.in (hunk 1 above)
Comment (fbergkemper) [reply webui/tests/regress/webui-bsock-connection-test-tls.in]: fixed

File: webui/tests/regress/webui-bsock-connection-test-tls.in (hunk 2 above)
Comment (fbergkemper) [reply webui/tests/regress/webui-bsock-connection-test-tls.in]: fixed

**Comment by franku** [comment]
merged into bareos-18.2 with commit 322ab5d48a8b8d280b1b0ddcb1615c75981c51d8

**Pull request closed without merging**